spring-cloud-dataflow-server/pom.xml

```diff
@@ -409,5 +415,15 @@
 				</pluginRepository>
 			</pluginRepositories>
 		</profile>
+		<profile>
+			<id>local-dev-oracle</id>
```

Comment:
We already have a precedent for this technique in [PRO for H2 dependency](https://github.com/pivotal/scdf-pro#running-locally-w-h2).